Happy Dance/Assets/_Scripts/GameTimer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Playables;
+
+using Liminal.Core.Fader;
+using Liminal.SDK.Core;
+
+/// <summary>
+/// GameTimer is sued to control the ending of the happy dance experience.
+/// </summary>
+public class GameTimer 
+    : MonoBehaviour
+{
+    public float SecondsLeft;
+    public float VolumeFadeSpeed;
+    public DanceDetector PlayerDanceDetector;
+    public PlayableDirector Director;
+
+    void Start()
+    {
+        Assert.IsNotNull(PlayerDanceDetector, "PlayerDanceDetector must not be null");
+        StartCoroutine(TimerCoro());
+    }
+
+	IEnumerator TimerCoro()
```

Comment:
tabs on this is all screwed up. 

Also, private IEnumerator

Reply:
Done

Reply:
I don't know how the tabs got there. I don't think I copied anything and my visual studio is set ot use spaces. :/